Dockerfile

```diff
@@ -23,6 +23,11 @@ FROM $base
 # Clean out the default to ensure we're using our updated content
 RUN rpm -e bootupd
 COPY --from=build /out/ /
+# Remove /var/roothome as workaround
```

Comment:
What error is this working around?

Reply:
https://github.com/bootc-dev/bootc/issues/1546